src/Pagination/PhalconFrameworkPaginatorAdapter.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * This file is part of the League\Fractal package.
+ *
+ * (c) Phil Sturgeon <me@philsturgeon.uk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace League\Fractal\Pagination;
+
+/**
+ * A paginator adapter for PhalconPHP/pagination.
+ *
+ * @author Thien Tran <fcduythien@gmail.com>
+ * 
+ */
+class PhalconFrameworkPaginatorAdapter implements PaginatorInterface
+{
+    /**
+     * A slice of the result set to show in the pagination
+     *
+     * @var \Phalcon\Paginator\AdapterInterface
+     */
+    protected $paginator;
+
+    /***
+     *
+     * @return void
+     */
+    public function __construct($paginator)
```

Comment:
This needs a type hint, and you can drop the doc block on it as it's not needed.